utilities/copy_files_to_nodes.py

```diff
@@ -0,0 +1,26 @@
+import os
+from keywords.constants import ENVIRONMENT_FILE
+from libraries.provision.ansible_runner import AnsibleRunner
+
+
+def create_files_with_content(content, sg_platform, sg_hostname, file_name, cluster_conf, path=None):
+    ansible_runner = AnsibleRunner(cluster_conf)
+    if path is None:
+        path = "/tmp/" + file_name
```

Comment:
does /tmp directory always exist? and if you need copying files to this directory, you will need to clean up, while you may have already done that in ansible script. 
is it possible to use platform tmp directory? such as every platform has it own tmp location and the OS is in charge of cleaning them up. 
just a discussion, not requesting any code change.